packages/tupaia-web/src/features/EnlargedDashboardItem/EnlargedDashboardVisual.tsx

```diff
@@ -120,6 +120,7 @@ export const EnlargedDashboardVisual = ({
       <TitleWrapper>
         <BackLink parentDashboardItem={parentDashboardItem} />
         {config?.name && <Title>{titleText}</Title>}
+        {config?.reference && <ReferenceTooltip reference={config.reference} />}
```

Comment:
This line is **the** change for this ticket. Everything else is just refactors I did while figuring out that this is what I was meant to do